### ISSUE TITLE
Melhoria de usabilidade na tela de cancelar inscrição

### ISSUE THREAD

**Issue by Pedro-Philipe** [redelivre/jaiminho#63]
Durante a tela de cancelar inscrição é necessário mais informações para que os usuários não se percam do que está acontecendo, como por exemplo uma mensagem de sucesso após clicar em cancelar inscrição. Tradução de labels e também a possibilidade de se inscrever nas listas.
![photo769748819543959580](https://cloud.githubusercontent.com/assets/15365573/20357216/36a31820-ac0e-11e6-9015-0802e5e87bbc.jpg)

**Comment by cabelotaina** [comment]
Legal pedro essas ideias, vou fazer algumas melhorias no shortcode que criamos para o jaiminho, acho que a ideia de se inscrever em outras listas é boa, @anapaulabrasa o que acha disso, ajuda a gente aqui para dar uma amadurecida na ideia, acho que consigo subir boa parte do que o pedro propos.

**Comment by anapaulabrasa** [comment]
Sim, essa é uma demanda importante. Penso que quanto mais objetiva essa tela, melhor. Por mim nem deveria apaecer a lista que a pessoa tá cadastrada e todas as outras, e sim apenas uma mensagem pra pessoa confirmar se realmente deseja se descadastrar. Aí ela clica confirmando e pronto. O que acha @jpmehl ?

De todo modo, se a escolha for manter essa tela aí,  abaixo da planilhazinha das listas dá pra por a mensagem "Confirme abaixo sua desinscrição".

E dentro da planilha trocar os seguintes termos: ao invés de "Assinados", usar "Inscrito"; e ao invés de "Cancelados" usar "Desinscrição" ou "Desinscrito".

Quanto ao lance de a pessoa poder se inscrever em outras listas, é importante lembrar que pra isso acontecer as outras listas precisam estar marcadas como públicas.
